lib/src/api/login_api_service.dart

```diff
@@ -0,0 +1,29 @@
+import 'dart:io';
+import 'package:flutter/material.dart';
+import 'package:http/http.dart' as http;
+import 'dart:convert';
+import 'package:shared_preferences/shared_preferences.dart';
+
+class APIService {
+  Future<Map> login(String userId, String password) async {
+    dynamic postBody = <String, String>{"userId": userId, "password": password};
+    String url = "http://52.136.112.220:57842/api/auth/login";
```

Comment:
Create a constant for `server address` url and then append it with your route. For eg here add constant `serverURL` to `libs/constants`. Import that constant here. then create a new constant `loginRoute` in this file, which has `serverURL+"/api/auth/login"`.

Reply:
Yes, I will do the needful in the next PR.